src/user/profile/ReadMe.js

```diff
@@ -0,0 +1,249 @@
+import React, { Component } from "react";
+import { Card, Button } from "react-bootstrap";
+import "./ReadMe.scss";
+import { Editor } from "@tinymce/tinymce-react";
+import { updateProfile, getProfile } from "../../actions/usersAction";
+import { connect } from "react-redux";
+
+class ReadMe extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      longDescription: true,
+      displayEditor: false,
+      currentText: "",
+      firstName: "",
+      lastName: "",
+      shortDesc: "",
+      location: "",
+      github: "",
+      fb: "",
+      linkedIn: "",
+      designation: "",
+      website: "",
+      longDesc: "",
+      canChangeName: "",
+    };
+  }
+
+  componentDidMount() {
+    this.props.getProfile();
```

Comment:
don't call any API just use info of profile component as a props as this is the child component of profile.js